### PR TITLE
Get rid of origin term in the code (close #58)

### LINKED ISSUE #58: Get rid of "origin" term in the code

### PR DESCRIPTION
**Pull request by churkin**
\cc @inikulin @miherlosev

### CODE REVIEW

**Comment by inikulin** [comment]
This is definitely not **ALL** usages
origin

\r-

File: src/processing/style.js [hunk 1]
```diff
@@ -33,17 +34,14 @@ class StyleProcessor {
...
 
             return this._replaceStylsheetUrls(css, url => {
-                var originUrlObj = parseProxyUrl(url);
+                var parsedProxyUrl = parseProxyUrl(url);
 
-                if (originUrlObj)
-                    return formatUrl(originUrlObj.originResourceInfo);
-
-                return url;
+                return parsedProxyUrl ? parsedProxyUrl.originUrl : url;
```
Comment (inikulin) [inline src/processing/style.js]: And we have `parsedProxyUrl.originUrl` again, WTF?

**Comment by churkin** [comment]
FPR

**Comment by inikulin** [comment]
lgtm

**Comment by miherlosev** [comment]
lgtm

**Pull request merged**